packs/src/tables/confusion.json

```diff
@@ -0,0 +1,80 @@
+{
+  "name": "Confusion",
+  "img": "icons/magic/air/wind-tornado-cyclone-purple-pink.webp",
+  "description": "",
+  "results": [
+    {
+      "type": 0,
+      "weight": 1,
+      "range": [
+        1,
+        1
+      ],
+      "drawn": false,
+      "documentCollection": "",
+      "_id": "JJjhhU1Br9ThNEWA",
+      "text": "The creature uses all its movement to move in a random direction. To determine the direction, roll a d8 and assign a direction to each die face. The creature doesn't take an action this turn.",
```

Comment:
You can convert the `d8` here to an inline roll.

Reply:
Changed this to an inline roll as suggested.